src/extension.ts

```diff
@@ -597,19 +597,14 @@ function launchMetals(
     // bar with a "cancel" button.
     client.onRequest(MetalsSlowTask.type, (params, requestToken) => {
       return new Promise((requestResolve) => {
+        const showLogs = ` ([show logs](command:${ClientCommands.toggleLogs} "Show Metals logs"))`;
```

Comment:
I was also thinking of `- show logs`, but not sure what is best

Reply:
This looks good! We can refine it later if it turns out to be weird